### ISSUE TITLE
Korabl-Sputnik-1 has requirement on non-existing 'R-7A' mission

### ISSUE THREAD

**Issue by rzabcio** [Frylovespi/History-of-Spaceflight#4]
It blocks whole Vostok program, and can create situation when Gemini is BEFORE Vostok.

~~~~
REQUIREMENT
{
          name = CompleteContract
          type = CompleteContract
          contractType = R-7A
          minCount =1
          maxCount =1
          cooldownDuration = 0d
}
~~~~

Should be probably:
~~~~
          contractType = R-7
~~~~

**Comment by Frylovespi** [comment]
Thank you. That is a left over from the patch that removed the R-7A mission. Looks like I missed one prereq to fix. I have deployed a fix on the github if you clone my mod you can get the latest fix. I will not have a release ready until I have time to squash a few more bugs but you can get it now. Or, you can go ahead and of course make the change in your own config file. You were correct in assuming it was supposed to be R-7.

TLDR: Fixed on github. Clone to download and install. Will be added to a hotfix release in the coming days
